conan_support.cmake

```diff
@@ -109,9 +109,19 @@ function(detect_host_profile output_file)
 endfunction()
 
 
+function(detect_conan_command)
+    if(DEFINED CONAN_COMMAND_PATHS)
+        find_program(CONAN_COMMAND "conan" PATHS ${CONAN_COMMAND_PATHS} REQUIRED NO_CMAKE_FIND_ROOT_PATH NO_DEFAULT_PATH)
+    else()
+        find_program(CONAN_COMMAND "conan")
```

Comment:
Is there any possibility that this will find a Conan different to the one that command line invocation would use? Like when being inside a virtualenv or something like that?

Reply:
If you provide paths suggestions it will find the conan that is present in those. E.g. if you install conan with apt and in an virtualenv you could not tell for sure which one will be found first without looking at the path. With this you can provide suggestions as to which one you want to use.